pkgs/servers/sql/postgresql/default.nix

```diff
@@ -99,6 +114,13 @@ let
     postPatch = ''
       # Hardcode the path to pgxs so pg_config returns the path in $out
       substituteInPlace "src/common/config_info.c" --replace HARDCODED_PGXS_PATH "$out/lib"
+
+      ${lib.optionalString jitSupport ''
+        # Force lookup of jit stuff in $out instead of $lib
+        substituteInPlace src/backend/jit/jit.c --replace pkglib_path \"$out/lib\"
+        substituteInPlace src/backend/jit/llvm/llvmjit.c --replace pkglib_path \"$out/lib\"
+        substituteInPlace src/backend/jit/llvm/llvmjit_inline.cpp --replace pkglib_path \"$out/lib\"
+      ''}
     '';
```

Comment:
```suggestion
      substituteInPlace "src/common/config_info.c" --replace HARDCODED_PGXS_PATH "$out/lib"
    '' + lib.optionalString jitSupport ''
      # Force lookup of jit stuff in $out instead of $lib
      substituteInPlace src/backend/jit/jit.c --replace pkglib_path \"$out/lib\"
      substituteInPlace src/backend/jit/llvm/llvmjit.c --replace pkglib_path \"$out/lib\"
      substituteInPlace src/backend/jit/llvm/llvmjit_inline.cpp --replace pkglib_path \"$out/lib\"
    '';
```

Reply:
I don't see the point in suggesting formatting changes like this.

Reply:
Nesting optionalString instead of appending them is not that nice and causes extra rebuilds.

Reply:
Can you suggest how inlining vs concating makes a difference regarding rebuilds?

Reply:
Sorry, my mistake! I confused this suggestion with the isDarwin-part below (where concatenation doesn't make sense).

In fact I already fixed this and because we don't have any trailing newlines in the drv here (in the previous version there was always one newline at the end of `postPatch` which caused a rebuild of the non-JIT packages on this branch which is why this used to point against staging), we actually saved some rebuilds (that's why this PR points to master again).